src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+// import PropTypes from 'prop-types';
+import { IsNotMarried } from '../Married/Married';
+import { PartnerName } from '../Married/PartnerName';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    {(!person.isMarried)
+      ? <IsNotMarried />
+      : <PartnerName sex={person.sex} name={person.partnerName} />
+    }
```

Comment:
That is an interesting approach 🤔 
But I believe it is better to keep it simple in that case, no need to create components for that